CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to the Mila Docs
+
+Thank you for your interest into making a better documentation for all at Mila. Here are some guidelines to help bring your contributions to life.
+
+## What could be included
+
+* Mila cluster usage
+* Compute Canada cluster usage
+* Job management tips / tricks
+* Research good practices
+* Software development good practices
+* Useful tools
+

```

Comment:
Fixing errors and typos

Reply:
Je vais renommer la section à _What should be included in the documentation_ mais je ne mettrais pas _Fixing errors and typos_ dans cette liste. Je pense que _Fixing errors and typos_ s'inclue dans la phrase plus bas:

>Issues can be used to report any error in the documentation, missing or unclear sections, broken tools or other suggestions to improve the overall documentation.

Reply:
Farpait. Merci.